python/ray/_private/worker.py

```diff
@@ -1283,6 +1283,9 @@ def init(
 
     if local_mode:
         driver_mode = LOCAL_MODE
+        warnings.warn(
+            "DeprecationWarning: local mode is an experimental feature that is no longer maintained and will be removed in the future."
```

Comment:
```python/ray/_private/worker.py:1287:89: E501 line too long (135 > 88 characters)```

Reply:
Can we unify the format of warning calls like:
```
       warnings.warn(
            "local mode is .....",
            DeprecationWarning,
            stacklevel=2,
        )
```